src/main/java/com/google/step/datamanager/RestaurantManagerDatastore.java

```diff
@@ -63,7 +63,9 @@ public Restaurant updateRestaurant(Restaurant restaurant) {
       restaurantEntity.setProperty("photoBlobkey", restaurant.photoBlobkey);
     }
     datastore.put(restaurantEntity);
-    return readRestaurant(restaurant.id);
+    String name = (String) restaurantEntity.getProperty("name");
+    String photoBlobkey = (String) restaurantEntity.getProperty("photoBlobkey");
+    return new Restaurant(restaurant.id, name, photoBlobkey);
```

Comment:
move these changes to PR #31

Reply:
i changed it back to readrestaurant which is the same as the other PR